source/Classes/020.RDConnectionBrokerHAMode.ps1

```
@@ -0,0 +1,159 @@
+        return ([ResourceBase] $this).Get()
+    }
+
+    [void] Set()
+    {
+        # Call the base method to enforce the properties.
+        ([ResourceBase] $this).Set()
+    }
+
+    [System.Boolean] Test()
+    {
+        $null = ([ResourceBase] $this).Test()
```

PSScriptAnalyzer (1.25.0)

[info] Ignoring 'TypeNotFound' parse error on type 'ResourceBase'. Check if the specified type is correct. This can also be due the type not being known at parse time due to types imported by 'using' statements.

(TypeNotFound)